shared/docs/android/running.md

```diff
@@ -0,0 +1,119 @@
+# Quick Start
+
+1. `adb devices` (should list *exactly* one device) 
+1. `adb reverse tcp:8081 tcp:8081` (port forward so react-native can reach its packager)
+1. `yarn rn-gobuild-android`  (build the go library)
+1. `yarn rn-start android` (start the packager)
+1. `yarn rn-build-android` (builds the apk. or `gradle installDebug` inside react-native/android)
```

Comment:
hunh, cool, maybe we can remove these yarn commands in favor of just telling people to do `gradle installDebug`?